### PR TITLE
Create public_dev_meetings.md

### PR DESCRIPTION
**Pull request by michelleellen**
# Gno.land Developer Call 

## Purpose

Gno.land Developer Call is a bi-weekly meeting held by Gno core devs and dev contributors to discuss technical issues, brainstorm, and collaborate on development work with regards to the smart contract platform. 

Currently, the calls are recorded and uploaded to Gno.land YouTube, however in the future they will be streamed to YouTube and/or Twitch for community members to listen to the live meeting. 

## Agenda

The agenda will be posted prior to the call and the dev contributors and larger community is welcome to add topics as long as they are technical in nature, and related to the Gno.land project. 

Ps: If you are new and interested in asking general questions, join our bi-weekly Gno.land Office Hours on Discord. 

## Attendance

The current attendees are the Gno.land core development team and core contributor community. We invite active contributors interested in joining the bi-weekly meeting to reach out and let us know how they are using Gno.land or how they want to support the engineering initiatives. 

## Upcoming meeting calendar

TODO: Add calendar feature

## Meeting archives

| Number | Date            | Agenda | Recording 
|--------|-----------------|--------|-----------
| 1      | March 8th, 2023 | [Agenda](https://github.com/gnolang/gno/issues/517) | Recording

### CODE REVIEW

File: public_dev_meetings.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Attendance
+
+The current attendees are the Gno.land core development team and core contributor community. We invite active contributors interested in joining the bi-weekly meeting to reach out and let us know how they are using Gno.land or how they want to support the engineering initiatives. 
+
+Upcoming meeting calendar
+
+TODO: Add calendar feature
+
+Meeting archives
```
Comment (zivkovicmilos) [inline public_dev_meetings.md]: You can add ### to this line to make it a Header 3:
`### Meeting archives`
```suggestion
### Meeting archives
```

File: public_dev_meetings.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+The agenda will be posted prior to the call and the dev contributors and larger community is welcome to add topics as long as they are technical in nature, and related to the Gno.land project. 
+
+Ps: If you are new and interested in asking general questions, join our bi-weekly Gno.land Office Hours on Discord. 
+
+Attendance
+
+The current attendees are the Gno.land core development team and core contributor community. We invite active contributors interested in joining the bi-weekly meeting to reach out and let us know how they are using Gno.land or how they want to support the engineering initiatives. 
+
+Upcoming meeting calendar
```
Comment (zivkovicmilos) [inline public_dev_meetings.md]: You can add ### to this line to make it a Header 3:
`### Upcoming meeting calendar`
```suggestion
### Upcoming meeting calendar
```

File: public_dev_meetings.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Currently, the calls are recorded and uploaded to Gno.land YouTube, however in the future they will be streamed to YouTube and/or Twitch for community members to listen to the live meeting. 
+
+
+Agenda
+
+The agenda will be posted prior to the call and the dev contributors and larger community is welcome to add topics as long as they are technical in nature, and related to the Gno.land project. 
+
+Ps: If you are new and interested in asking general questions, join our bi-weekly Gno.land Office Hours on Discord. 
+
+Attendance
```
Comment (zivkovicmilos) [inline public_dev_meetings.md]: You can add ### to this line to make it a Header 3:
`### Attendance`
```suggestion
### Attendance
```

File: public_dev_meetings.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+Gno.land Developer Call 
+
+Purpose
+
+Gno.land Developer Call is a bi-weekly meeting held by Gno core devs and dev contributors to discuss technical issues, brainstorm, and collaborate on development work with regards to the smart contract platform. 
+
+Currently, the calls are recorded and uploaded to Gno.land YouTube, however in the future they will be streamed to YouTube and/or Twitch for community members to listen to the live meeting. 
+
+
+Agenda
```
Comment (zivkovicmilos) [inline public_dev_meetings.md]: You can add ### to this line to make it a Header 3:
`### Agenda`
```suggestion
### Agenda
```

File: public_dev_meetings.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+Gno.land Developer Call 
+
+Purpose
```
Comment (zivkovicmilos) [inline public_dev_meetings.md]: You can add ### to this line to make it a Header 3:
`### Purpose`
```suggestion
### Purpose
```

File: public_dev_meetings.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+Gno.land Developer Call
```
Comment (zivkovicmilos) [inline public_dev_meetings.md]: You can add ## to this line to make it a Header 2:
`## Gno.land Developer Call`
```suggestion
## Gno.land Developer Call 
```

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯 

I've left a few suggestions

**Comment by michelleellen** [comment]
Here is the pull request @moul

**Comment by moul** [comment]
Thanks

I'll create a dedicated repo more focused on meetings and will come back to you.

**Comment by moul** [comment]
The new repo is https://github.com/gnolang/meetings. Please, open a new PR against the new repo.

**Pull request closed without merging**